dev/automated_tests/flutter_test/exception_handling_expectation.txt

```diff
@@ -46,14 +46,6 @@ Who lives, who dies, who tells your story\?
 When the exception was thrown, this was the stack:
 #[0-9]+ +main.<anonymous closure> \(.+[/\\]dev[/\\]automated_tests[/\\]flutter_test[/\\]exception_handling_test\.dart:16:5\)
 <<skip until matching line>>
-#[0-9]+ +.+ \(package:flutter_test[/\\]src[/\\]widget_tester\.dart:[0-9]+:[0-9]+\)
-<<skip until matching line>>
-#[0-9]+ +.+ \(package:flutter_test[/\\]src[/\\]binding.dart:[0-9]+:[0-9]+\)
-<<skip until matching line>>
-#[0-9]+ +.+ \(package:flutter_test[/\\]src[/\\]widget_tester\.dart:[0-9]+:[0-9]+\)
-<<skip until matching line>>
-^\(elided ([0-9]+|one) .+$
-<<skip until matching line>>
```

Comment:
I just removed these from the golden.  We don't actually care if it prints out this stuff, the important part is showing where the exception is thrown in the users code.